skglm/solvers/group_bcd_solver.py

```diff
@@ -119,3 +136,21 @@ def _bcd_epoch(X, y, w, Xw, datafit, penalty, ws):
             if old_w_g[idx] != w[j]:
                 Xw += (w[j] - old_w_g[idx]) * X[:, j]
     return
+
+
+@njit
+def _construct_grad(X, y, w, Xw, datafit, ws):
```

Comment:
same here.
I'd change the `-gradients are stacked...` docstring into a comment though.